person.rb

```diff
@@ -0,0 +1,37 @@
+require './nameable'
+
+class Person < Nameable
+  attr_reader :id
+  attr_accessor :name, :age
+
```

Comment:
- [ ] Kindly add Getters and Setters for @rentals, So the person class can access and create rentals.